templates/etc_init_archivematica-storage-service.conf.j2

```diff
@@ -5,4 +5,8 @@ author "Artefactual Systems"
 start on (runlevel [2345] and filesystem)
 stop on runlevel [016]
 
+env LANG="en_US.UTF-8"
+env LC_ALL="en_US.UTF-8"
+env LC_LANG="en_US.UTF-8"
```

Comment:
We already have a dictionary `archivematica_src_ss_environment` (defined [here](https://github.com/artefactual-labs/ansible-role-archivematica-src/blob/ee7785354cfd17290ff68f45df90237414532524/tasks/main.yml#L14-L30)) with our desired string environment, which is used in a number of places like gunicorn's [config file](https://github.com/artefactual-labs/ansible-role-archivematica-src/blob/ba682657bbaf6599d21137247b5d3dc5e8a7dc84/templates/etc_archivematica_storage-service.gunicorn-config.py.j2#L26-L31) or when invoke `manage.py` from Ansible. Would that be a better place too add your new items to make sure that they are applied consistently everywhere?